contrib/mpi-proxy-split/lower-half/libproxy.c

```diff
@@ -68,12 +68,22 @@ getDataFromMaps(const Area *text, Area *data, Area *heap)
       break;
     }
   }
-  // NOTE: Assume that data and heap are contiguous.
+  // NOTE: Assume that data, bss(if exists), and heap are contiguous.
+  /*
+   * The BSS section can be present on some platform and abscent on some.
```

Comment:
abscent -> absent on others.

Reply:
Also, can you update the comment and add a picture of the memory layout?

Reply:
For the picture, it's enough to give an example /proc/*/maps for a.out.  But can't we have four memory segments in the worst case, as below?
> ... r-x .../a.out # text segment
> ... r-- .../a.out  # read-only data segment
> ... rw- .../a.out # data segment
> ... rw- .../a.out # bss segment